gatsby-config.js

```diff
@@ -45,11 +45,11 @@ module.exports = {
             [
               'Content-Security-Policy:',
               'default-src \'self\';',
-              'script-src \'self\' data: \'unsafe-inline\' cdnjs.cloudflare.com www.googletagmanager.com tagmanager.google.com consent.cookiebot.com www.google-analytics.com cdn.cookielaw.org;',
+              'script-src \'self\' data: \'unsafe-inline\' cdnjs.cloudflare.com www.googletagmanager.com geolocation.onetrust.com tagmanager.google.com consent.cookiebot.com www.google-analytics.com cdn.cookielaw.org;',
```

Comment:
I still see `consent.cookiebot.com` here, should we remove it now?

Reply:
you are right this is not longer used, pr updated